test/dlc_tests/container_tests/bin/pytorch_tests/0001-test-adapt-BERT_pytorch-training-with-gloo-mpi-backend.patch

```diff
@@ -1,14 +1,14 @@
-From 041d5935dfdb0b3b0ee8ce38afbe528ffa29e58c Mon Sep 17 00:00:00 2001
-From: Your Name <you@example.com>
```

Comment:
Is this change required?

Reply:
After updating the patch to be applied to torch repo, I overwrote the entire fie, hence the change.